### PR TITLE
TECOPS-5891 Register Databricks schema

### PR DESCRIPTION
**Pull request by Andy-Hay**
Add registering of Databricks scheme/protocol so that it can be used in xo/dburl

### CODE REVIEW

File: pkg/conntest.go [hunk 1]
```diff
@@ -65,3 +65,36 @@ func query(db *sql.DB, driver string) (string, error) {
...
+	return "databricks://" + user + "@" + host + u.Path, nil
+}
+
+func GetProtocols(name string) []string {
+	return dburl.Protocols(name)
+}
+
+func SchemeDriverAndAliases(name string) (string, []string) {
+	return dburl.SchemeDriverAndAliases(name)
+}
```
Comment (peel) [inline pkg/conntest.go:100]: Nit: would it be worth to move everything related to Databricks into `databaricks.go` module to further organize the code?

File: pkg/conntest_test.go [hunk 2]
```diff
@@ -30,6 +30,35 @@ func TestMarshall(t *testing.T) {
...
+
+func isElementExist(s []string, str string) bool {
+	for _, v := range s {
+		if v == str {
+			return true
+		}
+	}
+	return false
+}
+
```
Comment (peel) [inline pkg/conntest_test.go:61]: Similarly, maybe it's worth moving into databaricks-specific module?

File: cmd/check.go [hunk 3]
```diff
@@ -33,6 +33,7 @@ var checkCmd = &cobra.Command{
 This application is a tool to generate the needed files
 to quickly create a Cobra application.`,
 	Run: func(cmd *cobra.Command, args []string) {
+		pkg.RegisterDatabricks()
```
Comment (peel) [inline cmd/check.go:36]: 🥳

**Review by peel — CHANGES_REQUESTED** [review]
I'm easily proven wrong on this, but it'd be cool to move the Databricks-specific code out of the base lib module and stick it into a new one.

File: pkg/conntest.go (hunk 1 above)
Comment (Andy-Hay) [reply pkg/conntest.go]: Extracted to its own file, made change much simpler :)

File: pkg/conntest_test.go (hunk 2 above)
Comment (Andy-Hay) [reply pkg/conntest_test.go]: Yes, same with this comment as well.  Put into own file, nicely simplifies the change.

**Pull request merged**